### PR TITLE
Import all old commands

### PR DESCRIPTION
**Pull request by ludeeus**
Source <https://raw.githubusercontent.com/awesome-automations/hassbot-data/master/commands.json>

### CODE REVIEW

File: data/discord_messages.yaml [hunk 1]
```diff
@@ -1,3 +1,1664 @@
 hello_world:
```
Comment (balloob) [inline data/discord_messages.yaml:1]: Shall we drop this one as we no longer need a placeholder ?

File: data/discord_messages.yaml [hunk 2]
```diff
@@ -1,3 +1,1664 @@
...
+
+    * HAOS & Supervised use [`ha core
+    check`](https://www.home-assistant.io/common-tasks/os/#configuration-check)
+
+    * Container uses [`docker`
+    commands](https://www.home-assistant.io/common-tasks/container/#configuration-check)
+
+    * Core requires you to [activate the venv
+    first](https://www.home-assistant.io/common-tasks/core/#configuration-check)
+list:
```
Comment (balloob) [inline data/discord_messages.yaml:1003]: This one can go

Comment (ludeeus) [reply data/discord_messages.yaml:1003]: We import all or nothing
70-80 % needs adjustments or can be removed.

File: data/discord_messages.yaml (hunk 1 above)
Comment (ludeeus) [reply data/discord_messages.yaml:1]: This is the only one that show a description

File: data/discord_messages.yaml [hunk 3]
```diff
@@ -1,3 +1,1664 @@
...
+  content: >-
+    [Secure Web Application Gateway](https://docs.linuxserver.io/general/swag)
+    aka SWAG - is the easiest way of setting up a reverse proxy server with SSL
+    (using Certbot and LetsEncrypt) for your Docker based install. See [their
+    documentation](https://docs.linuxserver.io/general/swag) on it, and if you
+    get stuck they also have [a Discord server](https://discord.gg/YWrKVTn) and
+    [forum](https://docs.linuxserver.io/general/swag).
+tts:
+  content: >-
+    If `tts` isn't working in your media players (and on a Google device at most
```
Comment (balloob) [inline data/discord_messages.yaml:1422]: @cogneato once this PR is merged, can you go through these snippets and update them to latest knowledge/UI. For example, this one needs to use a My link to point at https://my.home-assistant.io/redirect/network/ instead.

File: data/discord_messages.yaml [hunk 4]
```diff
@@ -1,3 +1,1664 @@
...
+  content: >-
+    When using Discord's _Reply_ feature it defaults to pinging the person you
+    reply to, which can get **very frustrating for the target**. Use _Shift +
+    click_ on the _Reply_ option, or click `@ ON` to `@ OFF` to stop this - on
+    the right side of the compose bar - you have to do this on every reply.
+
+
+    Or don't, and just wait for the recipient to get annoyed enough to block
+    you.
+raccoon:
```
Comment (balloob) [inline data/discord_messages.yaml:1522]: Can we hide commands from auto-complete?

And maybe we can also drop some. (in a future PR maybe). Do we really need bot commands to promote individual repo's?

Comment (ludeeus) [reply data/discord_messages.yaml:1522]: Yes
No

**Review by balloob — APPROVED** [review]

File: data/discord_messages.yaml (hunk 4 above)
Comment (ludeeus) [reply data/discord_messages.yaml:1522]: https://github.com/home-assistant/service-hub/commit/f6057187bfc0b5759653a09cdd3e4acb0680e8fe hides commands from auto complete if they are missing a description (all imported commands are currently missing that)

**Pull request merged**